app/services/service_provider_seeder.rb

```diff
@@ -41,8 +41,8 @@ def run
 
   def service_providers
     file = Rails.root.join('config', 'service_providers.yml').read
-    content = ERB.new(file).result
-    YAML.safe_load(content).fetch(rails_env)
+    file.gsub!('%%ENV%%', deploy_env) if deploy_env
```

Comment:
Did the `%%` syntax come from anywhere? We use `%{env}` in [identity-hostdata](https://github.com/18F/identity-hostdata/blob/37c66753ed5a98b953e7d6b3a11cb6e43130c6a9/spec/identity/hostdata/s3_spec.rb#L69), which also happens to map to a format Ruby handles natively (ex `format('%{env}', env: 'foobar')`) so I'm a little hesitant to invent a new syntax

Reply:
I made it up 😄. I don't mind `%{env}` so happy to switch to that.